CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
 
 - [#317](https://github.com/okta/okta-auth-js/pull/317) - `pkce` option is now `true` by default. `grantType` option is removed.
 
+- [#320](https://github.com/okta/okta-auth-js/pull/320) - `getWithRedirect`, `getWithPopup`, and `getWithoutPrompt` take a single options object
```

Comment:
Wouldn't a more informative note say what has been dropped?  If I'm passing two options today, and the CHANGELOG tells me I should only send one, I don't know which one.